expressAPIServer/app.js

```diff
@@ -1,48 +1,43 @@
-var createError = require('http-errors');
+// Filename: app.js
+// Description: Main file for API server. Connects to MySQL database and facilitates HTTP requests.
+
+// Load in NPM modules for express app and MySQL connection
 var express = require('express');
-var path = require('path');
-var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 let cors = require('cors');
+var mysql = require('mysql');
 
-var indexRouter = require('./routes/index');
-var usersRouter = require('./routes/users');
-
+// Create the express app
 var app = express();
 app.use(express.json());
 app.use(cors());
 
-/*
-// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  next(createError(404));
+// Connection to the database team Azure DB
+// Eventually we would like to replace this authentificaiton with tokens
+var config =
+{
+    host: 'classy-schedule-database.mysql.database.azure.com';,
+    user: 'db_test';,
+    password: 'fA!6#_&eaU9-EaeJ';,
+    database: 'db_dev';,
```

Comment:
should there be `;` here? when I copied this into my IDE it was giving me errors